packages/loader/driver-utils/src/utils.ts

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export const uint8ArrayToArrayBuffer = (array: Uint8Array): ArrayBuffer =>
+    array.buffer.slice(array.byteOffset, array.byteOffset + array.byteLength);
```

Comment:
Could you please open an issue to move this to common-utils? Does not need to block this PR, but we should move it out of this layer.
As part of move, I'd be great to generilize code in IsoBuffer.from(), where we do have 0x000 & 0x001 asserts today.

Also can it not do slice when array.byteOffset === 0 && byteLength === array.value.buffer.byteLength?

Reply:
https://github.com/microsoft/FluidFramework/pull/7013

Reply:
Created Pr to actually add it in common-utils. I will remove that from here(Driver-utils).